packages/typescript-estree/src/parser.ts

```diff
@@ -224,22 +231,21 @@ function applyParserOptionsToExtra(options: TSESTreeOptions): void {
 }
 
 function warnAboutTSVersion(): void {
-  if (
-    !isRunningSupportedTypeScriptVersion &&
-    !warnedAboutTSVersion &&
-    process.stdout.isTTY
-  ) {
-    const border = '=============';
-    const versionWarning = [
-      border,
-      'WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.',
-      'You may find that it works just fine, or you may not.',
-      `SUPPORTED TYPESCRIPT VERSIONS: ${SUPPORTED_TYPESCRIPT_VERSIONS}`,
-      `YOUR TYPESCRIPT VERSION: ${ACTIVE_TYPESCRIPT_VERSION}`,
-      'Please only submit bug reports when using the officially supported version.',
-      border,
-    ];
-    extra.log(versionWarning.join('\n\n'));
+  if (!isRunningSupportedTypeScriptVersion && !warnedAboutTSVersion) {
+    const isTTY = typeof process === undefined ? false : process.stdout.isTTY;
```

Comment:
In Prettier's build configuration, we have `process`, but not `process.stdout`.  🤷‍♂️

Reply:
darn...

probably just better to do what `parser` does, and clear the logger function:

https://github.com/typescript-eslint/typescript-eslint/blob/d4703e1a97760604cadb4167d5de4a897883119f/packages/parser/src/parser.ts#L87-L93

(note that this logger function isn't used anywhere else, it's only used to log the warning)